tests/test_date.py

```diff
@@ -637,6 +637,7 @@ def test_remove_year_in_russian(self):
         self.assertEqual(date.sanitize_date(u'2005г.'), u'2005 ')
         self.assertEqual(date.sanitize_date(u'2005 г.'), u'2005 ')
         self.assertEqual(date.sanitize_date(u'Авг.'), u'Авг')
+        self.assertEqual(date.sanitize_date(u'2019:'), u'2019')
```

Comment:
I don’t think `test_remove_year_in_russian` is the right test function for this change.